extensions/api/data-management/asset/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/AssetApiExtension.java

```diff
@@ -1,14 +1,15 @@
 /*
- * Copyright (c) 2022 ZF Friedrichshafen AG
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
```

Comment:
Maybe just list all the copyright holders in a comma separated list

Reply:
whups, thanks for pointing out. auto save actions...